### PR TITLE
Updates dependencies, test target

### PR DESCRIPTION
**Pull request by ashfurrow**
#236 updates the bundle but ~our CI infrastructure isn't working for open source contributors. I've opened this PR with the same commit to test.~ It turns out that our version of `danger-action` needed updating. See: https://github.com/MeilCli/danger-action/issues/80

~If this works, I'll also update the `Bootstrap` project, because the updated `Gemfile.lock` points to an updated CocoaPods version, which produces updated files in `pod install` 😅~ I've updated this.

Basically I deleted all the files in `ReferenceImages` and re-ran the tests to regenerate them. A lot of the images have file names dependent on the OS/device being ran on, so that's the cleanest way to go. I think I even deleted some that weren't being reference anymore 🤷

### CODE REVIEW

**Comment by joaolfp** [comment]
@ashfurrow There is an issues talking about [Workaround](https://github.com/Carthage/Carthage/issues/3201) for xcode 13

**Comment by ashfurrow** [comment]
Yaaaaay it's green. I've requested a review from @dogo just to get a double-check on my work. Once this PR gets merged in, @joaolfp's commit from #236 will also land in `master` so they'll get credit for their work 👍

**Comment by joaolfp** [comment]
Thanks @ashfurrow
